### DISCUSSION TITLE
My setup

### DISCUSSION THREAD

**Discussion by norfolk-fr** [mcguirepr89/BirdNET-Pi#702] [Show and tell]
Hello everyone, I discovered the project recently, I share with you my setup.
The raspberryPi 3b+ is installed in the garden shed, the microphone is fixed under the roof ridge.
The mic and the USB sound card are cheap and the whole thing works surprisingly well, many detections every day. I will try to fine tune the settings to get the best results.

Here is the link to my birdnet : [https://birdnet-drennec.ddns.net/](https://github.com/mcguirepr89/BirdNET-Pi/issues/url)

Thanks to all of you!
Alban

**Comment by lloydbayley** [comment]
Nice one! Welcome to the Bird-Pi'ers (not a real word but a little fun). I had a listen to a few and you have very petite, tweety birds there that are delightful to listen to. Well-done on your setup. Mine here in Australia is: [Here](https://berowra.birdnetpi.com/) and also on the main page of the README. Have a look under Best Recordings by Species and have a laugh with our Laughing Kookaburra!

**Reply by norfolk-fr** [reply]
Incredible, it sounds like monkeys :)
I get a permanent sound at around 8kHz, I have to troubleshot this behaviour.
Whish you nice day (or night:))

**Reply by lloydbayley** [reply]
It's not too bad as hum goes. It will be something to do with ground noise or a power supply. Try different power supplies, different mics (if you have one) and proximity to power sources or magnetic field generators is the way to go. It's very frustrating trying to diagnose it. Good luck!

**Comment by blogdemoi** [comment]
Ah, un français. I am in France too, I could hear your "Chouette Hulotte". My Birdnet-Pi is near Paris: [https://birdnetpi.arnauld.org/](https://birdnetpi.arnauld.org/)

**Reply by norfolk-fr** [reply]
Hey ! nice to meet you. I can access your birnet-pi but it says "no detection for today" at 09:15am and I saw an error message about the database, a backup should be running maybe.
Have a nice day in Garches :)
